rules-reviewed/openshift/tests/local-storage.windup.test.xml

```diff
@@ -9,7 +9,7 @@
                 <when>
                     <not>
                         <iterable-filter size="14">
-                            <hint-exists message="Access to local storage on Openshift platform should be minimized and wherever possible replaced by using.*"/>
+                            <hint-exists message=".*Local files in containers are not permanent storage.*" />
```

Comment:
This can be triggered for below tests too, so it won't be stable test for this hint